src/Content/Nav.php

```diff
@@ -309,8 +309,8 @@ private function getInfo(): array
 			if ($this->session->get('page_flags', '') != User::PAGE_FLAGS_COMMUNITY) {
 				$nav['introductions']         = ['notifications/intros', $this->l10n->t('Introductions'), '', $this->l10n->t('Friend Requests')];
 				$nav['notifications']         = ['notifications',	$this->l10n->t('Notifications'), '', $this->l10n->t('Notifications')];
-				$nav['notifications']['all']  = ['notifications/system', $this->l10n->t('See all notifications'), '', ''];
-				$nav['notifications']['mark'] = ['', $this->l10n->t('Mark as seen'), '', $this->l10n->t('Mark all system notifications as seen')];
+				$nav['notifications']['all']  = ['notifications/system?show=all', $this->l10n->t('View all'), '', ''];
+				$nav['notifications']['mark'] = ['', $this->l10n->t('Mark as seen'), '', ''];
```

Comment:
Why have you removed the tooltips?

Reply:
Alright :+1: , that change has been reverted.